webhook_to_fedora_messaging/endpoints/models/service.py

```diff
@@ -0,0 +1,57 @@
+from abc import ABC
+from datetime import datetime
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class ServiceBase(BaseModel, ABC):
+    """
+    Base: Service
+    """
+
+    class Config:
+        from_attributes = True
+
+
+class ServiceExternal(ServiceBase):
+    uuid: str
+    name: str
+    type: str
+    desc: str
+    token: str
+    user_id: int
+    creation_date: datetime
+
+
+class ServiceInternal(ServiceExternal):
+    id: str
+
+
+class ServiceRequestMain(BaseModel):
+    name: str
+    type: str
+    desc: Optional[str]
+
+
+class ServiceRequest(BaseModel):
+    data: ServiceRequestMain
+
+
+class ServiceUpdateMain(BaseModel):
+    name: Optional[str] = ""
+    type: Optional[str] = ""
+    desc: Optional[str] = ""
+    user_uuid: Optional[str] = ""
```

Comment:
It's called `user_uuid` here but `user_id` in `ServiceExternal`. We should pick one. Since we talked about getting rid of the user's UUID, let put the `user_name` here (and in `ServiceExternal`), OK?